src/Build/mkdocs.jl

```diff
@@ -0,0 +1,94 @@
+function makemkdocs(build)
+    site_name = "Docile.jl"
+    repo_url = "https://github.com/MichaelHatherly/Docile.jl"
+    site_description = "Julia package documentation system."
+    site_author = "Michael Hatherly"
+    theme = "readthedocs"
+    docs_dir = joinpath(splitdir(Base.source_dir())[end], build)
```

Comment:
> or just a dict

Yeah, if that would also work then I'm fine with that as well.

> raise if they're not provided

We could just provide defaults if they're not given for the ones that make sense to have defaults for.

> one issue here is the ordering: alphabetical. Is, most of the time, not what you want. :s

Not sure what to do about that. Could just leave it up to the user to write that part themselves.